contact.html

```diff
@@ -16,7 +15,9 @@
 	</head>
 	
 	<body>
-		
+	

```

Comment:
This background image was trialled to see if it would look good in the background of this contact page, I found it looked better, so I used it